index.html

```diff
@@ -0,0 +1,156 @@
+            <div class="contact-us">Please contact us per Email for any further questions about The Youth Submit 2023!</div>
+            <div class="email">youthsummit2023@gmail.com</div>
+    </section>
+    <div class="logo-explanation">Youth Leadership Summit 2023 Logo</div>
+         <hr class="hr1 hr21">
+         <div class="explanation">The logo of Youth Leadership Summit 2023 was decided through the logo competition from 8, June to 7, July.</div>
+         <p class="explanation2">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+         <div class="summit"><img src="images/images/about_logo 1.png" alt="loading"></div>
+         <div class="summit-logo"><img class="img-fluid" src="images/images/about_logo 1 (1).png" alt="loading"></div>
+         <hr class="hr3 hr33">
+         <hr class="hr3">
+         <div class="past-summits">See the past Youth Summits</div>
+         <hr class="hr101">
+
+         <div class="look-in-past">Take a look at the past two Youth summits which took place in Moscow and Canada.</div>
+         <ul class="pictures">
+            <li><div class="group"><img class="img-fluid" src="images/images/about_2011 1.png" alt="loading"></div></li>
+            <li><div class="group-of-3"><img class="img-fluid" src="images/images/about_2013 1.png" alt="loading"></div></li>
+         </ul>
+         <footer>
+            <div class="partners">Partner
+                <ul class="partner-list">
+             <li><img src="images/images/mozilla 1.png" alt="loading"></li>
+                    <li><img src="images/images/googlekorea 1.png" alt="loading"></li>
+                    <li><img src="images/images/naver 1.png" alt="loading"></li> 
+                    </ul>
+                    <ul class="partner-list partner-list2">
+                    <li class="block"><img src="images/images/daumkakaologo 1.png" alt="loading"></li>
+                    <li class="block"><img src="images/images/airbnb 1.png" alt="loading"></li>
+                </ul>
+            </div>
+         </footer>
+         <footer>
+              <!-- Desktop footer -->
+            <div class="partners2" id="sponsors">Partner
+                <div class="partners2">Partner</div>
+                <hr class="partner-underline">
+                <div class="desk-partners"><img class="img-fluid" src="images/images/partners_logo.png" alt="loading"></div>
+            </div>
+         </footer class="desk-footer">
+         <!-- End of Deskyop footer -->
```

Comment:
- [ ] As requested by the previous reviewer you should create 2 pages, About and Home page. Currently, you only have one and it seems you also added some of the content of the about page in the home page. That's not how it should be. You are meant to create another HTML file about.html and implement the content t of the about page there. Also, remember to add the link to the About page in the nav list in your main navigation.
### Index and about page on same page.
![his](https://github.com/denigit5/First-Capstone/assets/81727988/5f69d46f-347f-4439-bd00-9f79b2c510bd)